### PR TITLE
Youtube Settings to remove distracting elements and recommended

### PR DESCRIPTION
**Pull request by sujitacharya2005**
Resolves https://github.com/brave/brave-browser/issues/27095

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [x] Checked the PR locally:
  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`
- [x] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

## Test Plan:

image

https://user-images.githubusercontent.com/32419898/229124631-31b0fa20-a1c3-49b4-9adf-61fd24a5ac04.mp4

### CODE REVIEW

File: android/java/org/chromium/chrome/browser/privacy/settings/BravePrivacySettings.java [hunk 1]
```diff
@@ -118,6 +124,8 @@ public class BravePrivacySettings extends PrivacySettings implements ConnectionE
             PREF_BRAVE_SOCIAL_BLOCKING_SECTION, // social blocking section
             PREF_SOCIAL_BLOCKING_GOOGLE, PREF_SOCIAL_BLOCKING_FACEBOOK,
             PREF_SOCIAL_BLOCKING_TWITTER, PREF_SOCIAL_BLOCKING_LINKEDIN,
+            PREF_YOUTUBE_SECTION, // Youtube section
+            PREF_HIDE_YOUTUBE_RECOMMENDED_CONTENT, PREF_HIDE_YOUTUBE_DISTRACTING_ELEMENTS,
```
Comment (deeppandya) [inline android/java/org/chromium/chrome/browser/privacy/settings/BravePrivacySettings.java:128]: need for check format for `PREF_HIDE_YOUTUBE_DISTRACTING_ELEMENTS`

File: browser/ui/android/strings/android_brave_strings.grd [hunk 2]
```diff
@@ -368,6 +368,23 @@ This file contains all "about" strings.  It is set to NOT be translated, in tran
       <message name="IDS_BRAVE_SHIELDS_BLOCKS_SCRIPTS_SUMMARY" desc="Switch for Block Scripts summary.">
         Blocks JavaScript (may break sites)
       </message>
+
```
Comment (deeppandya) [inline browser/ui/android/strings/android_brave_strings.grd:371]: we should remove this new line.

**Review by deeppandya — APPROVED** [review]
LGTM. amazing work. just minor comments which should be addressed before merging the change.

File: android/java/org/chromium/chrome/browser/privacy/settings/BravePrivacySettings.java (hunk 1 above)
Comment (sujitacharya2005) [reply android/java/org/chromium/chrome/browser/privacy/settings/BravePrivacySettings.java:128]: Not getting format specific error.
npm run format
works fine

File: browser/ui/android/strings/android_brave_strings.grd (hunk 2 above)
Comment (sujitacharya2005) [reply browser/ui/android/strings/android_brave_strings.grd]: updated

**Pull request merged**